lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -707,7 +707,11 @@ public function computeChangeSet(ClassMetadata $class, $entity)
                 continue;
             }
 
-            $this->computeAssociationChanges($assoc, $val);
+            try {
+                $this->computeAssociationChanges($assoc, $val);
+            } catch (\Exception $ex) {
```

Comment:
Not sure how i can track all the possible situations which yield all possible Exceptions